addon/-private/utils/element.js

```diff
@@ -30,13 +30,45 @@ export function closest(el, selector) {
   return null;
 }
 
+/*
+ * Calculate the scale of difference between an element's logical height
+ * (the offsetHeight or getComputedStyle height) compared to its rendered
+ * height (via getBoundingClientRect).
+ *
+ * Note that there are some interesting edge cases to consider around
+ * these APIs.
+ *
+ *   - `getComputedStyle` returns a string from styles in pixels. For
+ *     example `48.23px`. The precision of this API in Chrome seems to
+ *     be 3 decimal places. Additionally, there can be unexpected values
+ *     such as `auto`. Finally, this API may be slow compared to other
+ *     DOM API calls.
+ *   - `offsetHeight` always returns in integer. If the height of an
+ *     element is at float precision, then the value is rounded. This
+ *     makes it at best useful as a fallback.
+ *
+ */
 export function getScale(element) {
   let rect = element.getBoundingClientRect();
+  let renderedHeight = rect.height;
 
-  if (element.offsetHeight === rect.height || rect.height === 0) {
+  if (renderedHeight === 0) {
+    return 1;
+  }
+
+  let computedHeightStyleValue = window.getComputedStyle(element).height;
```

Comment:
do we have a performance test for rendering a large table (100 visible rows) with dynamic height ? Or maybe you have done it manually, if automated cannot catch slowdowns ?

because i am pretty sure you are aware of the dangers (killing performance) with `getComputedStyle`, but will still put an article (not THE article, just AN article) link here:

https://pagebuildersandwich.com/increased-plugins-performance-200/

Reply:
I've addressed this. `getScale` is now only called once and is passed through the system where needed.